fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/ImportHandlerUtils.java

```diff
@@ -63,8 +63,13 @@ public static Long readAsLong(int colIndex, Row row) {
         FormulaEvaluator eval = row.getSheet().getWorkbook().getCreationHelper().createFormulaEvaluator();
         if(c.getCellType() == Cell.CELL_TYPE_FORMULA) {
             if(eval!=null) {
-                CellValue val = eval.evaluate(c);
-                return ((Double) val.getNumberValue()).longValue();
+                CellValue val = null;
```

Comment:
Can you please let us know in which scenario a null pointer exception is thrown.  As there is a null check for  'c and eval', I am not able to figure out  the case when a null pointer exception is thrown  by the statement in try block . This applies for all remaining method changes.

Reply:
@ShruthiRajaram When any excel formula is evaluated to null, a null pointer is thrown fromApache POI API, this is what's being handled.

Reply:
A known issue in the previous version of data import tool and fixed by PR in the description

Reply:
Referring https://github.com/openMF/DataImportTool/pull/34